test/e2e/autotls/nonhttp01/auto_tls_test.go

```diff
@@ -0,0 +1,133 @@
+// +build e2e
```

Comment:
Tests in this file are directly copied from https://github.com/knative/serving/blob/master/test/e2e/autotls/auto_tls_test.go

So it is not necessary to review this file.